run_unittests.py

```diff
@@ -6338,11 +6338,10 @@ def test_identity_cross_env(self):
         testdir = os.path.join(self.unit_test_dir, '61 identity cross')
         env = {
             'CC_FOR_BUILD': '"' + os.path.join(testdir, 'build_wrapper.py') + '"',
+            'CC': '"' + os.path.join(testdir, 'host_wrapper.py') + '"',
         }
         crossfile = tempfile.NamedTemporaryFile(mode='w')
-        crossfile.write('''[binaries]
-c = ['{0}']
-'''.format(os.path.join(testdir, 'host_wrapper.py')))
+        crossfile.write('')
```

Comment:
I don't have to change this test, as this is a non-breaking change. However, doing so makes it less redundant with the previous test, which exclusive uses the native and cross files with no environment variables.